apps/ima/views/harm-claim-summary.html

```diff
@@ -0,0 +1,16 @@
+{{<partials-page}}
+  {{$page-content}}
+   <br>
+        {{> partials-harm-claim-summary-table}}
+        <p>If you have included a British citizen who is your child, you must upload their birth certificate later in this form.</p>
```

Comment:
This paragraph is not in the Figma design. Is it needed?

Reply:
Paragraph removed